product-service/serverless.ts

```diff
@@ -0,0 +1,55 @@
+import type { Serverless } from 'serverless/aws';
+
+const serverlessConfiguration: Serverless = {
+  service: {
+    name: 'product-service',
+    // app and org for use with dashboard.serverless.com
+    // app: your-app-name,
+    // org: your-org-name,
+  },
+  frameworkVersion: '2',
+  custom: {
+    webpack: {
+      webpackConfig: './webpack.config.js',
+      includeModules: true
+    }
+  },
+  // Add the serverless-webpack plugin
+  plugins: ['serverless-webpack'],
+  provider: {
+    name: 'aws',
+    runtime: 'nodejs12.x',
+    apiGateway: {
+      minimumCompressionSize: 1024,
+    },
+    environment: {
+      AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
+    },
+  },
+  functions: {
+    products: {
+      handler: 'handler.getProductsList',
+      events: [
+        {
+          http: {
+            method: 'get',
+            path: 'products',
+          }
+        }
+      ]
+    },
+    product: {
+      handler: 'handler.getProductsById',
+      events: [
+        {
+          http: {
+            method: 'get',
+            path: 'product',
```

Comment:
Why don't you use path params?
```suggestion
            path: 'product/{id}',
```